lib/generators/framework/templates/framework_definition.rb.erb

```diff
@@ -0,0 +1,29 @@
+<%
+  def exports_to(field)
```

Comment:
I wonder if this work would be better done outside of the template file? maybe moving it into the generator somehow, either as a helper, or by having the generator create an appropriate data structure so that the field loops below don't have to have this conditional logic in them.

Reply:
Yes I think we can use a `private` method in the ERB `binding`, I'll try that.